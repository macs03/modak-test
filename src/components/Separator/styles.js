import { StyleSheet } from "react-native";

const styles = StyleSheet.create({
  separator: {
    // marginVertical: 30,
    height: 1,
    width: "95%",
    backgroundColor: "#aaa",
    alignSelf: "center"
  }
});

export default styles;
