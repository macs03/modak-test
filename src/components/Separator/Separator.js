import { View } from "react-native-ui-lib";
import styles from "./styles";

const Separator = () => {
  return (
    <View
      style={styles.separator}
      lightColor="#eee"
      darkColor="rgba(255,255,255,0.1)"
    />
  );
};

export default Separator;
