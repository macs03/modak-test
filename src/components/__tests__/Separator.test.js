import * as React from "react";
import renderer from "react-test-renderer";

import Separator from "../Separator/Separator";

it(`renders correctly`, () => {
  const tree = renderer.create(<Separator />).toJSON();

  expect(tree).toMatchSnapshot();
});
