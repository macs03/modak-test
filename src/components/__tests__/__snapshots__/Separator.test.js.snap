// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  darkColor="rgba(255,255,255,0.1)"
  lightColor="#eee"
  style={
    Array [
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
      Object {
        "alignSelf": "center",
        "backgroundColor": "#aaa",
        "height": 1,
        "width": "95%",
      },
    ]
  }
/>
`;
